demos/common/models/src/detection_model_retinaface.cpp

```diff
@@ -0,0 +1,391 @@
+/*
+// Copyright (C) 2018-2020 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+*/
+
+#include <ngraph/ngraph.hpp>
+#include <samples/common.hpp>
+#include <samples/slog.hpp>
+#include "models/detection_model_retinaface.h"
+
+using namespace InferenceEngine;
+
+ModelRetinaFace::ModelRetinaFace(const std::string& modelFileName, float confidenceThreshold, bool useAutoResize, float boxIOUThreshold)
+    : DetectionModel(modelFileName, confidenceThreshold, useAutoResize, {"Face"}),  // Default label is "Face"
+    boxIOUThreshold(boxIOUThreshold), maskThreshold(0.8f), shouldDetectMasks(false), landmarkStd(1.0f),
+    anchorCfg({ {32, { 32, 16 }, 16, { 1 }},
+              { 16, { 8, 4 }, 16, { 1 }},
+              { 8, { 2, 1 }, 16, { 1 }} }) {
+    generateAnchorsFpn();
+}
+
+void ModelRetinaFace::prepareInputsOutputs(InferenceEngine::CNNNetwork& cnnNetwork) {
```

Comment:
You already have `using namespace InferenceEngine`. Remove `InferenceEngine::` or `using namespace InferenceEngine;`.

Reply:
Removed using namespace.